x-pack/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/model/rule_assets/prebuilt_rule_asset.ts

```diff
@@ -37,6 +36,5 @@ export const PrebuiltRuleAsset = BaseCreateProps.and(TypeSpecificCreateProps).an
     version: RuleVersion,
     related_integrations: RelatedIntegrationArray.optional(),
     required_fields: RequiredFieldArray.optional(),
-    setup: SetupGuide.optional(),
```

Comment:
Thanks 👍

Reply:
This is in reference to [this comment](https://github.com/elastic/kibana/pull/179680#issuecomment-2051874416), so yes if you're adding those fields to `BaseDefaultableFields` fields, they should respectively be removed in this type.

Reply:
Got it. Removed "required_fields" in my required fields PR.